### ISSUE TITLE
Add ability to encrypt/decrypt data

### ISSUE THREAD

**Issue by itzen** [roikku/swift-explorer#44]
I would be nice to have functionality to:
- Encrypt data before send it to storage,
- Decrypt data after download it from storage.

**Comment by COLABORATI** [comment]
Yes, this is the elephant in the room. I would go one step further and would like to ask you to recognize the current "encrypt everything" effort and add client site encryption as a default operation mode.

BTW I would like you to add at least to the FAQ some note about the fact that this software is NOT encrypting anything that is uploaded. Even better and some kind of fair would be to put this as a warning somewhere everybody can read it clearly, e.g. on a dialog box that appears on start of the software  - the current state of not even mentioning the word "encryption" anywhere is not acceptable. 

Please add, as long as you are working on this feature, a warning on the frontpage of your site about your software not being secure for private data, thanks!
